packages/core/src/tracing/instrumentation/messaging/rdkafka.js

```diff
@@ -340,42 +349,7 @@ function addTraceContextHeader(headers, span) {
   if (!traceCorrelationEnabled) {
     return headers;
   }
-  switch (headerFormat) {
-    case 'binary':
-      return addTraceContextHeaderBinary(headers, span);
-    case 'string':
-      return addTraceContextHeaderString(headers, span);
-    case 'both':
-    // fall through (both is the default)
-    default:
-      return addTraceContextHeaderBinary(addTraceContextHeaderString(headers, span), span);
-  }
-}
```

Comment:
I've removed all logic for adding binary headers from rdkafka, since they basically do not work anyway.